CONTRIBUTING.md

```diff
@@ -12,45 +12,57 @@ Pro tip: Use the table of contents in the top-left corner to jump to specific se
+
+(team member) If you need help, feel free to consult your development lead or other members of the team. 
 
-### Creating Issues
 
-To create an issue as a member of our team, locate the appropriate template for your issue from the [new issue screen](https://github.com/hackforla/CivicTechJobs/issues/new/choose). Typically, you will not go wrong with [Blank Issue](https://github.com/hackforla/CivicTechJobs/issues/new?assignees=&labels=&template=blank-issue.md&title=) or the [form version](https://github.com/hackforla/CivicTechJobs/issues/new?assignees=&labels=&template=blank-issue-form.yml). If you need help, feel free to consult your development lead or other members of the team. Once you are finished, [add your newly opened issue to our Project Management project board](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar).
 
 ### Resolving Issues
 
 > **Note: Make sure to check out our [installation instructions](https://hackforla.github.io/CivicTechJobs/developer/installation/) to set up your development environment before working on any issues!**
 
 1. Find a developer issue to work on from our [prioritized backlog](https://github.com/hackforla/CivicTechJobs/projects/1#column-10928271). (Hint: look for the `frontend` or `backend` label). The `size` label will signal the amount of work the issue might involve, with a higher number indicating greater work. For a development team member, we encourage taking the topmost `frontend` or `backend` issue, as they are arranged by priority by role.
-2. Assign yourself to the issue. (Hint: see [step 4](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users#assigning-an-individual-issue-or-pull-request).)
-3. Move the issue from the prioritized backlog to the [in progress](https://github.com/hackforla/CivicTechJobs/projects/1#column-10928272) column. (Hint, see [step 5](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar).)
+2. Indicate you are working on the issue as follows:
+- (team member) Assign yourself to the issue. (Hint: see [step 4](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users#assigning-an-individual-issue-or-pull-request).)
+- (freelancer) If you expect the issue will take more than a day to complete, add a comment to the bug indicating you are working on the bug 
```

Comment:
The formatting of these lines look weird when the markdown is rendered. Let's indent them with a couple of spaces and see how they render.